.github/workflows/docs.yml

```diff
@@ -0,0 +1,52 @@
+name: Docs
+
+on: [push, pull_request]
+
+
+jobs:
+  publish-docs:
+    name: Build and publish documentation
+    if: github.repository_owner == 'NSLS-II'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9"]
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Set up Python ${{ matrix.python-version }} with conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: spcc-py${{ matrix.python-version }}
+          auto-update-conda: true
+          miniconda-version: "latest"
+          python-version: ${{ matrix.python-version }}
+          
+      - name: Build example code
+        shell: 'script --return --quiet --command "bash -l {0}"'
+        run: |
+            set -vxeo pipefail
+            conda env list
+            pip install -r requirements.txt
+            ./run_cookiecutter_example.py
+            pip install -e example/
+            ./copy_user_content.sh
+            pytest example/
+            
+      - name: Build docs
+        shell: bash -l {0}
+        run: make -C docs html
+        
+      - name: Publish docs
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' && matrix.python-version == '3.9' }}
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.ACTIONS_DOCUMENTATION_DEPLOY_KEY }}
+          publish_branch: master
+          publish_dir: ./docs/build/html
+          external_repository: NSLS-II/NSLS-II.github.io
+          destination_dir: ${GITHUB_REPOSITORY#*/}  # just the repo name, without the "NSLS-II/"
```

Comment:
The `GITHUB_REPOSITORY` var should be set in the very beginning, something like https://github.com/NSLS-II/sirepo-bluesky/blob/9e9d78cb346650f71c2960ff62580bb24ee4daf4/.github/workflows/publish-docs.yml#L25-L26.

Reply:
GITHUB_REPOSITORY is available to all steps, so this should work.

Reply:
I haven't been able to verify this will work so I have adopted the sirepo-bluesky method.